internal/controller/reconcile-capapplicationversion.go

```diff
@@ -25,7 +25,6 @@ import (
 	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
(thought I already pushed this last time)
this is already imported as metav1 above